tools/openmrs_setup

```diff
@@ -71,6 +71,33 @@ mysql -uroot -p <<< "
 "
 echo 'Added MySQL user "openmrs_user" with password "openmrs".'
 
+# Running from Linux or Mac? Set the appropriate path.
+buendia_dir='/usr/share/buendia'
+bin_dir='/usr/bin'
+OS=`uname`
+if [[ "$OS" == 'Darwin' ]]; then
+    buendia_dir='/usr/local/opt/buendia'
+    bin_dir='/usr/local/bin'
+fi
+profiles_dir="$buendia_dir/profiles"
+
+# TODO: remove the need for sudo
+# Create directories and copy scripts
+sudo mkdir -p $profiles_dir
```

Comment:
You should echo a message informing that the folder xxx will be create.
If not, the user will not even know what password he/she is supposed to fill. 

<img width="613" alt="screen shot 2015-10-23 at 6 45 39 pm" src="https://cloud.githubusercontent.com/assets/665045/10704754/e103e8b0-79b7-11e5-9bb3-b6fae771b726.png">

As you can see, the script prompts for the mysql password and then it asks again for a password. What for ?